cmd/curio/cli.go

```diff
@@ -253,3 +257,87 @@ var waitApiCmd = &cli.Command{
 		return ctx.Err()
 	},
 }
+
+var indexSampleCmd = &cli.Command{
+	Name:      "index-sample",
```

Comment:
I feel like this is a really specialized command which possibly will be grouped with other commands at some point. Probably fine now, but if we get more related commands here we should group them together into `index` or `data` subcommand